samples/gradle/.mainframerignoreremote

```diff
@@ -0,0 +1,5 @@
+.gradle
```

Comment:
`gradle-android` will make more sense since our userbase mostly consists of Android developers (for now).